lib/helpers.ts

```diff
@@ -55,8 +54,8 @@ export interface BaseSerializableEntity {
     dataset_id: string;
     level: string;
     assayName?: string;
-    primaryParents?: DataFileID[];
-    synapseId?: string;
+    primaryParents?: synapseId[];
```

Comment:
There is a mismatch here. It should be like this:

```suggestion
    primaryParents?: SynapseId[];
```